content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst

```diff
@@ -1,90 +1,123 @@
+   <inventory/shipping/invoice-so>`
 
-2.  Invoice the real cost of the shipping.
+#. :ref:`Invoice shipping costs to the customer post-delivery
+   <inventory/shipping/invoice-shipping>`, reflecting the actual expenses incurred by the business.
 
 Configuration
 =============
 
-To configure the price of your delivery methods, go to the **Inventory**
-app, click on :menuselection:`Configuration --> Delivery --> Delivery Methods`.
+To set prices to delivery methods, go to :menuselection:`Inventory app --> Configuration -->
+Settings`. Under the :guilabel:`Shipping` section, enable the :guilabel:`Delivery Methods` feature.
+Then, click :guilabel:`Save`.
 
-You can manually set a price for the shipping: It can be fixed or based
-on rules.
+.. image:: invoicing/enable-delivery.png
+   :align: center
+   :alt: Enable the "Delivery Methods" feature in Settings.
 
-Or you can use the transportation company computation system by :doc:`integrating a third-party
-shipper <../setup_configuration/third_party_shipper>`.
+Add shipping method
+===================
 
-How to invoice the shipping costs to the customer?
-==================================================
+Next, configure the the price of each delivery method by going to :menuselection:`Inventory app -->
+Configuration --> Shipping Methods` and click the :guilabel:`Create` button. Doing so opens a form
+to provide details about the shipping provider, including:
 
-Invoice the price set on the sale order
----------------------------------------
+- :guilabel:`Shipping Method` (*required*) the name of the delivery method (e.g. `flat-rate
+  shipping`, `same day delivery`, etc.).
+- :guilabel:`Provider` (*required*): choose the delivery service, like FedEx, if using a
+  third-party carrier Ensure the integration with the shipping carrier is properly installed and
+  select the provider from the drop-down menu.
 
-On your sale order, choose the carrier that will be used. Click on
-**Delivery Method** to choose the right one.
+  .. seealso::
+     :doc:`../setup_configuration/third_party_shipper`
 
-.. image:: invoicing/invoicing06.png
-   :align: center
+- :guilabel:`Company`: if the shipping method should apply to a specific company, select it from the
+  drop-down menu. Leave the field blank to apply the method to all companies.
 
-The price is computed when you **save** the sale order or when you click on
-**Set price**.
+- :guilabel:`Website`: configure shipping methods for an e-commerce page. Select the applicable
+  website from the drop-down menu, or leave it blank to apply the method to all web pages.
 
-To invoice the price of the delivery charge on the sale order, click on
-**Set price**, it will add a line with the name of the delivery method as
-a product. It may vary from the real price.
+- :guilabel:`Delivery Product` (*required*): the product listed on the :ref:`sales order line
+  <inventory/shipping/sales-order>` as the delivery charge.
```

Comment:
@Felicious I had to remove the ref (`inventory/shipping/sales-order` added [line 81 in the 15.0 PR](https://github.com/odoo/documentation/blob/ba49211c55c067b818fbe659b234b6cb68e787cc/content/applications/inventory_and_mrp/inventory/shipping/operation/invoicing.rst?plain=1#L81)) because that exact ref already exists in the 16.0 documentation. 

Can you please verify that the [existing 16.0 ref](https://github.com/odoo/documentation/blob/8a0d3598a62b454ceb8817de4bf4d7e9ad6bed41/content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/delivery_method.rst?plain=1#L171) is still accurate for this FWP PR?

Reply:
Instead of removing the duplicate ref from this FWP PR, I will change the custom anchor declaration to differ...